Detectors/PHOS/calib/src/PHOSL1phaseCalibDevice.cxx

```diff
@@ -41,19 +41,46 @@ void PHOSL1phaseCalibDevice::run(o2::framework::ProcessingContext& pc)
   auto cellTR = pc.inputs().get<gsl::span<TriggerRecord>>("cellTR");
   LOG(detail) << "Processing TF with " << cells.size() << " cells and " << cellTR.size() << " TrigRecords";
   mCalibrator->process(tfcounter, cells, cellTR);
+
+  ++mNprocessed;
+  // If sufficient statistics was collected, try to calculate L1phases
+  // If statistics reached N*thresholds try to re-calculate L1phases
+  // if they changed wrt previous, replace ccdb object, otherwise do nothing
+  if (mNprocessed % minStatisticsForCalib == 0) {
+    minStatisticsForCalib *= 2;
```

Comment:
Note that you update the calibration not at `N*trhesholds` but at `2^(N-1) * thresholds`. Why do you need to `minStatisticsForCalib *= 2;` after each update?

Reply:
L1 object is unique for whole run. Normally it should be calculated from minimal threshold. If in the run we have noisy channels, I think it is better to collect considerably more statistics before next try. This should reduce the number of overlapped objects in ccdb.

Reply:
ok, thanks, though then the comment about `N*thresholds`  is somewhat misleading...